File: frontend/src/models/InventoryProductDTO.js
export class InventoryProductDTO {
   constructor(productId, warehouseId, name, description, price, quantity) {
       this.productId = productId;
       this.warehouseId = warehouseId;
       this.name = name;
       this.description = description;
       this.price = price;
       this.quantity = quantity;
   }
}

File: backend/src/main/java/teamx/app/backend/models/dto/InventoryProductDTO.java
package teamx.app.backend.models.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class InventoryProductDTO {
    private Long productId;
    private Long warehouseId;
    private String name;
    private String description;
    private Double price;
    private int quantity;
}
